core/src/test/resources/log4j.properties

```diff
@@ -19,7 +19,6 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
 log4j.logger.kafka=ERROR
-log4j.logger.org.apache.kafka=ERROR
```

Comment:
Probably unintentional.

Reply:
Yes. I will fix the log4j.properties as we near the end of the review process.